packages/engine/Source/Core/ITwin.js

```diff
@@ -0,0 +1,306 @@
+ * @property {number} minLineStyleComponentSize
+ */
+
+/**
+ * @typedef {Object} ViewDefinitionFilter
+ * @property {string[]} models Array of included model IDs.
+ * @property {string[]} categories Array of included category IDs.
+ * @property {string[]} neverDrawn Array of element IDs to filter out.
+ */
+
+/**
+ * @typedef {Object} StartExport
+ * @property {string} iModelId
+ * @property {string} changesetId
+ * @property {ExportType} exportType Type of mesh to create. Currently, only GLTF and 3DFT are supported and undocumented CESIUM option
+ * @property {GeometryOptions} geometryOptions
+ * @property {ViewDefinitionFilter} viewDefinitionFilter
+ */
+
+/**
+ * @typedef {Object} Link
+ * @property {string} href
+ */
+
+/**
+ * @typedef {Object} Export
+ * @property {string} id
+ * @property {string} displayName
+ * @property {ExportStatus} status
+ * @property {StartExport} request
+ * @property {{mesh: Link}} _links
+ */
+
+/**
+ * @typedef {Object} ExportResponse
+ * @property {Export} export
+ */
+
+/**
+ * Default settings for accessing the iTwin platform.
+ *
+ * Keys can be created using the iModels share routes {@link https://developer.bentley.com/apis/imodels-v2/operations/create-imodel-share/}
+ *
+ * An ion access token is only required if you are using any ion related APIs.
+ * A default access token is provided for evaluation purposes only.
+ * Sign up for a free ion account and get your own access token at {@link https://cesium.com}
+ *
+ * @see IonResource
+ * @see IonImageryProvider
+ * @see IonGeocoderService
+ * @see createWorldImagery
+ * @see createWorldTerrain
+ * @namespace ITwin
+ */
+const ITwin = {};
+
+/**
+ * Gets or sets the default iTwin access token.
+ *
+ * TODO: I'm not sure we can even do this kind of access token. Each route seems to need it's own scopes
+ * and we may not be able to guarantee this "top level token" has them all
+ * So far we use
+ * `mesh-export:read` for loading meshes GET /mesh-export(s)
+ * `mesh-export:modify` if we want to include a function to create an export
+ * `itwin-platform` if we want to use the iModel shares ourselves  GET /imodels/{id}/shares
+ *
+ *
+ * @type {string|undefined}
+ */
+ITwin.defaultAccessToken = undefined;
+
+/**
+ * Gets or sets the default iTwin API endpoint.
+ *
+ * @type {string|Resource}
+ * @default https://api.bentley.com
+ */
+ITwin.apiEndpoint = new Resource({
+  url: "https://api.bentley.com",
+});
+
+/**
+ * @param {string} exportId
+ */
+ITwin.getExport = async function (exportId) {
+  if (!defined(ITwin.defaultAccessToken)) {
+    throw new DeveloperError("Must set ITwin.defaultAccessToken first");
+  }
+
+  const headers = {
+    Authorization: ITwin.defaultAccessToken,
+    Accept: "application/vnd.bentley.itwin-platform.v1+json",
+  };
+
+  // obtain export for specified export id
+  const url = `${ITwin.apiEndpoint}mesh-export/${exportId}`;
+
+  // TODO: this request is _really_ slow, like 7 whole second alone for me
```

Comment:
fyi @danieliborra @arvindsvenkat

Reply:
I was occasionally seeing this jump to ~20-23 seconds. It seemed very repeatable that I got one of those two timings for the [Get export](https://developer.bentley.com/apis/mesh-export/operations/get-export/) route specifically. Interestingly the [Get exports](https://developer.bentley.com/apis/mesh-export/operations/get-exports/) route for the full list is really fast and almost always under 1 second, usually under 200-300ms too even with `return=representation`.

Reply:
That is because `get-export` is not very used and so we haven't optimized it yet. All apps use `get-exports`, and we have kept `get-export` to avoid breaking changes.
This quarter we will clean-up the API.

Reply:
@danieliborra In the meantime, would using `get-exports` be preferred then?